pyanaconda/modules/common/structures/partitioning.py

```diff
@@ -34,7 +34,7 @@ def __init__(self):
         self._excluded_mount_points = []
         self._hibernation = False
 
-        self._encrypted = False
+        self._encrypted = conf.storage.encrypted
```

Comment:
Same question - what does this do outside Qubes OS?

Reply:
Similar to hostname, it has no impact for other profiles because it uses default values from `anaconda.conf` where I've put the original values that was in the code.

Reply:
Thanks!